src/Entity/OgMembership.php

```diff
@@ -58,13 +59,21 @@
 class OgMembership extends ContentEntityBase implements ContentEntityInterface {
 
   /**
+   * @param array $values
+   *
+   * @return OgMembership
+   */
+  public static function create(array $values = []) {
+    return parent::create($values + ['type' => OgMembershipInterface::TYPE_DEFAULT]);
```

Comment:
I thought base fields can have defaults set on them? Or is this necessary because type is the bundle and the entity system goes bonkers if you try to create without a bundle? I guess a core issue is in order to cover the "default bundle" case.